index.html

```diff
@@ -20,6 +20,12 @@
       <!-- header section  -->
       <header class="header">
          <a href="#" class="logo">John Doe</a>
```

Comment:
Great job @John-Kibirige 🤝

- [ ] You may please personalize this logo to your own name. This is your portfolio website so it doesn't need to contain any other person's initials.